modules/system/controllers/Settings.php

```diff
@@ -187,4 +217,23 @@ protected function findSettingItem($author, $plugin, $code)
 
         return $item;
     }
+
+    /**
+     * Auto-guesses the requested setting item by URL segments from the Request object.
+     *
+     * @return array
+     */
+    protected function autoGuessSettingItem()
+    {
+        $segments = Request::segments();
+
+        if (!empty(Config::get('cms.backendUri', 'backend'))) {
```

Comment:
Have you tested it with both `/backend` and `/`?

Reply:
Yep, confirmed it working with `/` as well.